scenario/gasSchedules/gasScheduleV4.toml

```diff
@@ -225,6 +225,9 @@
     UnmarshalCompressedECC = 270000
     GenerateKeyECC = 7000000
     EncodeDERSig = 10000000
+VerifySecp256r1 = 2000000
```

Comment:
Alignement issue ^
+ are the values ok to be equal for different types of verifications?

Reply:
this still seems to be here

Reply:
done